store/migration/dev/20230117000000##idp.sql

```diff
@@ -0,0 +1,25 @@
+-- idp stores generic identity provider.
```

Comment:
We only need one migration file instead of two separate.

Reply:
merged